### PR TITLE
Update dependency versions for update 6

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by tharindu-nw — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 org.gradle.caching=true
 group=io.ballerina
-version=0.5.1-SNAPSHOT
+version=0.6.0-SNAPSHOT
 
 #dependency
 ballerinaLangVersion=2201.5.0
```
Comment (NipunaRanasinghe) [inline gradle.properties:6]: Don't we need to update the ballerina lang timestamped version?

Comment (dilanSachi) [reply gradle.properties:6]: No need right. Since this doesnt depend on the latest lang changes we dont need to

Comment (NipunaRanasinghe) [reply gradle.properties:6]: But the other stdlibs timestamps are released on top of the lang update 6 timestamp IINM. so shouldn't we update the lang timestamp as well?

**Review by NipunaRanasinghe — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (dilanSachi) [reply gradle.properties:6]: Other stdlibs have not update their lang versions as well.

**Pull request merged**